### DISCUSSION TITLE
Request: Delete sensor values from the database

### DISCUSSION THREAD

**Discussion by FredD35** [hassio-addons/addon-influxdb#254] [Ideas]
Hello,

as time goes by and HA is nicely storing values in the Influx DB, the amount of data is constantly growing. The more entities we add to the system, the more is stored in the DB. 
If we are honest, most of the data is not really needed, but it is pretty nice to see how long a specific light bulb has been switched on the same night two years ago.
Yes, we are able to configure in HA, what should be stored, but normally we tend to forget about it, once a new device has been introduced. And many devices have a longer list of attributes, so also a longer list of values is stored.
Then we have values in the db, where we've just tried something out, like creating a sensor and in the end, we needed several tries until it worked like expected - but before it works, several footprints have been left in the DB. Not everyone runs a test system next to the productive system.

Deleting values in the influx DB is possible, but also not the easiest part. Searching google leads to many hits and in the end, it boils down to "I would like to delete values from sensor A - how can I do this".

So my question is, would it be a proper feature request for the Influx HA addon to provide a small UI to be able to enter the name of the entity and the attributes to be deleted? Maybe even a time span as well, but this could be something for version two.

Best regards,
Fred

**Comment by vahaldor** [comment]
That would be really great. I have been trying to delete sudden erroneous spike in energy consumption and despite all the help around including the templates provided by influxdb GUI, I am not successful. For those, who work with databases on a daily basis this feature seems to be unnecessary, but for us, the HA users, this would be a jewel.
